src/test/java/org/cactoos/text/StrictTest.java

```diff
@@ -38,6 +40,21 @@
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
 final class StrictTest {
 
+    /**
+     * Ensures that Strict accepts a CharSequence predicate.
+     */
+    @Test
+    void acceptsCharSequencePredicate() {
+        final Func<CharSequence, Boolean> longerThanThree = new FuncOf<>(
+            seq -> seq.length() > 3
+        );
+        new Assertion<>(
+            "Given strings are not equal",
```

Comment:
@DmitryBarskov please phrase it with "Must..." like all the other tests in cactoos :)